hack/util.sh

```diff
@@ -485,12 +485,21 @@ function dump_container_logs()
 	done
 }
 
-# delete_large_and_empty_logs deletes empty logs and logs over 20MB
-function delete_large_and_empty_logs()
-{
+# delete_empty_logs deletes empty logs
+function delete_empty_logs() {
 	# Clean up zero byte log files
+	find "${ARTIFACT_DIR}" "${LOG_DIR}" -type f -name '*.log' \( -empty \) -delete
+}
+
+# truncate_large_logs truncates large logs so we only download the last 20MB
+function truncate_large_logs() {
 	# Clean up large log files so they don't end up on jenkins
-	find "${ARTIFACT_DIR}" "${LOG_DIR}" -type f -name '*.log' \( -empty -or -size +20M \) -delete
+	local large_files=$(find "${ARTIFACT_DIR}" "${LOG_DIR}" -type f -name '*.log' \( -size +20M \))
+	for file in "${large_files}"; do
+		cp "${file}" "${file}.tmp"
+		tail -c 20M "${file}.tmp" > "${file}"
```

Comment:
tack a header on the log file indicating it was truncated so people aren't baffled later as to why it's cut off.